processor/configuration/src/main/java/com/redhat/service/smartevents/processor/InternalKafkaConnectionPayloadImpl.java

```diff
@@ -0,0 +1,28 @@
+package com.redhat.service.smartevents.processor;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.redhat.service.smartevents.processor.actions.kafkatopic.KafkaTopicAction;
+
+@ApplicationScoped
+public class InternalKafkaConnectionPayloadImpl implements InternalKafkaConnectionPayload {
+
+    @Inject
+    GatewayConfiguratorService gatewayConfiguratorService;
+
+    @Override
+    public void addInternalKafkaConnectionPayload(String bridgeId, String processorId, ObjectNode actionParameters) {
```

Comment:
Not needed anymore?

Reply:
When we'll have a common class for connector action it won't be needed but currently yes